src/client/components/contact-form/contact-form.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <form
+    v-if="ownerEmail"
+    :action="`https://formspree.io/${ ownerEmail }`"
+    class="contact-form"
+    method="POST">
+    <input type="hidden" name="_cc" :value="userEmail" />
+    <input type="hidden" name="_next" :value="nextUrl" />
+    <input type="hidden" name="_subject" :value="form.title" />
+    <input type="hidden" name="_language" :value="$i18n.locale" />
+    <input type="hidden" name="_gotcha" style="display:none" />
+
+    <h2 class="email-form__title">{{ form.title }}</h2>
+    <rich-text v-if="form.body" :text="form.body" class="contact-form__body" />
+
+    <rich-text :id="messageId" :text="form.confirmationMessage" class="alert alert--success contact-form__alert" />
+
+    <label class="contact-form__label" :for="`${formId}-message`">{{ form.labelMessage }}</label>
+    <textarea class="contact-form__field form-control" placeholder="Your message..." :id="`${formId}-message`" name="message" required></textarea>
+
+    <label class="contact-form__label" :for="`${formId}-name`">{{ form.labelName }}</label>
+    <input class="contact-form__field form-control" :id="`${formId}-name`" type="text" placeholder="Your name" name="name">
```

Comment:
If you call these elements `form-control` I would expect the contact form specific class to be `contact-form__control`. And in general I think it is more common to have this as a structure:
```
FORM FIELD
  FORM LABEL
  FORM CONTROL
```
This way the controls can have a small default margin bottom and the fields can have additional margin to space those out.